article/sections/_2_graph_importance.tex

```diff
@@ -0,0 +1,189 @@
+\section{Perils of disregard}
+\label{sec:graph-importance}
+
+As stated in \ref{sec:intro}, the development of transportation demand models aims at evaluating the impact of policies on a certain
+transportation demand related outcome. As an example, consider the proposals from the following fictitious scenarios: 
+- Based on input from the public, the Department of Transportation (DOT) of  is considering implementing a new parking policy. 
+This proposed policy would put in place pricing changes and parking restrictions that would discourage individuals from parking in the central business district. 
+The DOT (and its constituents) believes that this policy would encourage people to use more active transportation modes (e.g: walking, biking, etc..).
```

Comment:
@hassan-obeid does this address your [comment on line 15](https://github.com/hassan-obeid/tr_b_causal_2020/pull/150/files/c65814292b16eae3444b434c69fad2ecca151074?file-filters%5B%5D=.bib&file-filters%5B%5D=.pdf&file-filters%5B%5D=.png&file-filters%5B%5D=.tex&file-filters%5B%5D=No+extension#r532124593)

```suggestion
The DOT (and its constituents) believes that this policy would encourage people to use more active transportation modes (e.g: walking, biking, etc.) and cause them to drive less.
```